src/spyglass/lfp/v1/analytic_signal.py

```diff
@@ -0,0 +1,129 @@
+import datajoint as dj
+import numpy as np
+import pandas as pd
+from scipy.signal import hilbert
+import math
+from spyglass.utils.dj_helper_fn import fetch_nwb
+from spyglass.lfp.v1 import LFPBand, LFPElectrodeGroup
+from spyglass.common.common_interval import IntervalList
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+
+schema = dj.schema("lfp_v1_analytic_signal")
+
+
+@schema
+class AnalyticSignalParameters(dj.Manual):
```

Comment:
Right now an error would be thrown when populating the AnalyticSignal table when using other methods than "hilbert_transform". I think that should be sufficient? But otherwise, how to make an overloaded insert1?

Reply:
Nvm it's solved I added an overloaded inser1 :)